src/components/Game/functions/playGame.js

```diff
@@ -0,0 +1,127 @@
+export const playGame = tiles => {
+  console.clear();
+  let currentTileY = 0;
+  let currentTileX = 0;
+  let entry = 0;
+  let exit = 0;
+  let finishLevel = false;
+
+  const playGameNextTile = (y, x, entry) => {
```

Comment:
We should have a call an go through this one together so you can take me through it, is the game working?